examples/test/animate-color/index.html

```diff
@@ -0,0 +1,79 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8">
+    <title>Animate Color</title>
+    <meta name="description" content="Animate Color - A-Frame">
+    <script src="../../../dist/aframe.js"></script>
+  </head>
+  <body>
+    <a-scene stats>
+      <a-assets>
+        <a-mixin id="box" class="pulse" geometry="primitive: box; depth: 0.5; height: 1; width: 0.5" material="color: #5698Af">
+        <a-mixin id="pulse" attribute="material.color" from="#fff" to="#5698Af" dur="1000" begin="pulse"></a-mixin>
+      </a-assets>
+
+      <a-sky color="#000000">
+        <a-animation attribute="color" dur="10000" from="#afafaf" to="#000000" repeat="indefinite">
+      </a-sky>
+
+      <a-entity position="4.5 1 -3" class="pulse" mixin="box"><a-animation mixin="pulse"></a-animation></a-entity>
```

Comment:
You don't need the class anymore because your `selectorAll` can be `[mixin="box"]`